### ISSUE TITLE
Gscan task/job summary content

### ISSUE THREAD

**Issue by hjoliver** [cylc/cylc-ui#615]
March meeting decision: gscan should show:
 - *main state icon list*: 
      - summary of n=0 tasks (active + unhandled-failed) 
 - *state icon tool-tips*:
     -  number of active tasks of that state, 
     -  and the 5 most recent tasks to have entered that state (which may include tasks that are no longer active)

Remaining questions:
1. n=0 includes (for now) tasks with satisfied task-dependencies but still waiting on hold, queue, or xtrigger. To represent these in the list of gscan task/job icons should we show:
      - a single "waiting" task icon?
      - or separate hold, queued, xtriggered indicator badges
2. sometimes there are no active tasks of a particular state, but we'll still want to show the "5 most-recent" via tool-tip. How do we do this?
      -  always keep all (the active + waiting) icons in the gscan list, to host the tool-tips, but make them very greyed out when there are no active tasks of that state?

**Comment by hjoliver** [comment]
Note I'm assuming that we do want gscan to show (e.g.) number of queued tasks. That is useful information (if a flow is not very active but there are loads of queued tasks, you can see at a glance the reason for the low activity)

**Comment by jarich** [comment]
A note for when gscan shows all running workflows, rather than just the users' workflows.

The list of workflows across all other users might be quite long, with easily tens to potentially even low hundreds of workflows available. It should be possible to see a good proportion of the existing workflows within a single window, either by grouping (as per the gtk interface) or otherwise.  It might be nice if it were possible to open the current scan pane into a separate window, and potentially even add extra columns to display scanned suites more densely through width rather than only length.

**Comment by hjoliver** [comment]
Closed by #617 (with possible further tweaks to come).

@jarich - I'll transfer your comment to the multi-user Issue.

**Comment by hjoliver** [comment]
> I'll transfer your comment to the multi-user Issue.

Actually we haven't got one of those yet on cylc-ui (I was thinking of the UIS).  Making one...
